docs/metrics/ga.md

```diff
@@ -173,19 +191,21 @@ We use custom dimensions to refine our pageviews on Test Pilot ([docs from Googl
-| Experiment Detail Page                        | Experiment title                                  | 5         | "xyz"  |
-| Experiment Detail Page                        | Installation count                                | 6         | {n}    |
-| Home Page, Experiment Detail Page             | Determine if installation will require a restart  | 7         | {'no restart','restart required'}   |
-| Home Page, Experiment Detail Page, Share Page | Which test has this user been selected for?       | 8         | {'installButtonBorder', ''} |
-| Home Page, Experiment Detail Page, Share Page | Which cohort has this user been selected for?     | 9         | {'bigButton', ''} |
-| All                                           | In which responsive breakpoint is the user?       | 10        | {'mobile','small','medium','big'} |
-| All experiment-specific locations and events  | On which experiment did this action take?         | 11        | experiment slug | 
+| Page                                          | Description                                             | dimension | values |
+|-----------------------------------------------|---------------------------------------------------------|-----------|--------|
+| Home Page, Experiment Detail Page, Share Page | Does the user have the add-on installed                 | 1         | {0,1}  |
+| Home Page, Share Page                         | Does the user have any experiments installed            | 2         | {0,1}  |
+| Home Page, Share Page                         | How many experiments does the user have installed       | 3         | {n}    |
+| Experiment Detail Page                        | Is the experiment enabled                               | 4         | {0,1}  |
+| Experiment Detail Page                        | Experiment title                                        | 5         | "xyz"  |
+| Experiment Detail Page                        | Installation count                                      | 6         | {n}    |
+| Home Page, Experiment Detail Page             | Determine if installation will require a restart        | 7         | {'no restart','restart required'}   |
+| Home Page, Experiment Detail Page, Share Page | Which test has this user been selected for?             | 8         | {'installButtonBorder', ''} |
+| Home Page, Experiment Detail Page, Share Page | Which cohort has this user been selected for?           | 9         | {'bigButton', ''} |
+| All                                           | In which responsive breakpoint is the user?             | 10        | {'mobile','small','medium','big'} |
+| All experiment-specific locations and events  | On which experiment did this action take?               | 11        | experiment slug |
+| Featured Experiment Video Progress            | Amount of video watched (0:<=10%, 1:10%>x<90%, 2:>=90%) | 12        | {0,1,2}|
```

Comment:
My only suggestion here is to use names rather than numbers, which will make reporting in GA much more satisfying.